data/hai/hai reveal 1 intro.txt

```diff
@@ -564,11 +566,11 @@ mission "Hai Leaks Response 2"
 	passengers 4
 	on offer
 		event "pirates respond to leaks"
-		log "People" "Xilin Yang" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and recent peace.`
+		log "People" "Yang Xilin" `Deputy Minister Yang works in the Foreign Affairs department of the Republic government. He is a pleasant and astute man. Handling the Hai secrecy crisis has become his job while the majority of the foreign ministry remains engaged with the fallout from the Free Worlds and recent peace.`
```

Comment:
Yang is his last name, this is how Chinese names work, and why you usually hear for Xi Jinping, "President Xi", instead of "President Jinping". Jinping is the first name. I commented earlier: 

> By the way, Chinese names go surname:first name, so Xilin's proper full name would be Yang Xilin. I don't think this would have changed in a thousand years between now and Endless Sky. 

Deputy Minister Yang is still correct, and calling him Xilin is also correct. There should be no conflict with his name in the rest of the story.

Reply:
I know, and maybe it's a regional thing, but most of the Chinese I've known (english/spanish/french speaking country here) reverse it, using the western convention since westerners will institutionally call them by their first name. If people call everyone in your family by the same name, it gets confusing when talking to strangers (mother, father, sister, and dog are all Zhang). The humans in Endless Sky will probably have the same issue since all of the characters thus far have used the First Last ordering.

Reply:
My tone might have been too harsh, haha.

This may be a regional thing. The city I live in (not close to China, but around 5-10% population is Chinese), and the family I have, will always use last:first. If they have to, they'll also have english names, but among themselves they will always use their Chinese name. Otherwise, people seem to get around it fine.

Reply:
Yeah it's very situationally dependent on how integrated they are with other cultures I find.

This particular name is based on a colleague of mine who uses their name in the way it's used here.
The subtle implication being that he isn't particularly close to his roots.

Reply:
With that being the explicit intent, I think reverting it is the best option.

Without that though, I do think the change was the best course.

Another thing I'd like to see hashed out (later, in a separate PR) then, though, is some kind of passing reference to reinforce that implication.